boba_community/boba-node/docker/Dockerfile.data-transport-layer

```diff
@@ -1,5 +0,0 @@
-FROM bobanetwork/data-transport-layer:release
-
-COPY boba-node/docker/state-dump.latest.json /opt/optimism/packages/data-transport-layer/state-dumps/state-dump.latest.json
```

Comment:
We need insert the state-dump.latest.json to the dtl, so the l2geth can load the genesis block

Reply:
Also, the geth can't start if it can't read the state-dump.latest.json from dtl. [geth.sh](https://github.com/omgnetwork/optimism-v2/blob/develop/ops/scripts/geth.sh)
```
curl \
    --fail \
    --show-error \
    --silent \
    --retry-connrefused \
    --retry-all-errors \
    --retry $RETRIES \
    --retry-delay 5 \
    $ROLLUP_STATE_DUMP_PATH \
    -o genesis.json
```